data/STRchive-loci.json

```diff
@@ -4711,9 +4711,9 @@
   "typ_age_onset_max": null,
   "novel": "ref",
   "mechanism": "LoF",
-  "mechanism_detail": "Proposed mechanism involves repeat expansions in non-coding regions of the gene, reducing expression in melanocytes or keratinocytes, leading to a disruption in nucleotide balance in DNA repair and hyperpigmentation [@doi:10.3892/br.2025.2016].",
+  "mechanism_detail": "Proposed mechanism involves repeat expansions in non-coding regions of the gene, reducing expression in melanocytes or keratinocytes, leading to a disruption in nucleotide balance in DNA repair and hyperpigmentation. Missense mutations disrupt nucleotide metabolsim, resulting in loss-of-function and genome instability [@doi:10.3892/br.2025.2016].",
   "source": [],
-  "details": "Missense mutations disrupt nucleotide metabolsim, resulting in loss-of-function and genome instability [@doi:10.3892/br.2025.2016].",
+  "details": "A study has identified an intronic GATGGT hexanucleotide tandem repeat in the TYMS gene. Both parents were found to be heterozygous carriers of the expansion, suggesting a recessive inheritance pattern [@doi:10.3892/br.2025.2016].",
```

Comment:
One more request. I think it needs to be made clear that there is limited evidence. I suggest adding that:
Only a single family with two monozygotic twin probands has been reported.
No change in expression of the gene has been observed so the variant has yet to be biochemically confirmed as the causal variant. Or something along those lines.